README.md

```diff
@@ -3,56 +3,145 @@
+```js
+// Here we manually create a url with all the url parameters
+var url = 'https://api.spotify.com/v1/search?q=Take+Me+to+Church&type=track&limit=10';
+
+var success_callback = function (searchResultsFromAPIRequest){
+  var tracks = searchResultsFromAPIRequest.tracks.items;
+  $.each(tracks, function(index, track) {
+    console.log("Song Found: " + track.name);
+  });
+};
+
+// No url parameters being passed since we already included them in the url
+$.getJSON(url, success_callback);
+```
+In the end, we get the same results but we have the option to choose our approach based on the situation. One final thing to consider is that our success callback won't actually be called if the JSON that gets return from the API is invalid. We will need to have a error callback to handle these types of problems.
 
+## Instructions
+You will need to visit https://github.com/settings/tokens and create a personal token to use before we begin. This token allows you to make requests for data related to your account. Once you have this token we can continue.
 
-## Resources
+Copy the following code into `js/api_client.js`.
+```js
+var printStargazers = function(users) {
+  $.each(users, function(index, user) {
+    console.log(user.login + ' starred the Rails Repository');
+  });
+};
+```
 
+This function will print an array of users to the console. Next, copy the following code into `js/api_client.js`.
 
+```js
+  $.ajax({
+    url: 'https://api.github.com/repos/rails/rails/stargazers',
+    type: 'POST',
+    dataType: 'jsonp'
+  }).done(function(response) {
+    var users = response.data;
+    printStargazers(users);
+  });
+```
+This code makes a request to the GitHub API for all the users that starred the Ruby on Rails repository. Notice the dataType we provided was `jsonp`. This allows us to make requests for JSON to APIs that might not allow it otherwise. The callback calls the `printStargazers` function. Let's try out our code so far. In your terminal run `python -m SimpleHTTPServer`. Browse to http://localhost:8000 and open Chrome developer tools. You should see a number of logs like this `dhh starred the Rails Repository`.
 
+Now let's try a POST request. Copy the following code into `js/api_client.js`. 
+
+```js
+  var addHTML = function (html){
+    $('#search_results').html(html);
+  };
+  
+  var bindCreateButton = function (){
+    $('#convert').click(function(event) {
+      $.ajax({
+        url: 'https://api.github.com/markdown',
+        type: 'POST',
+        data: '{ "text": "#Hello world!", "mode": "markdown" }'
+      }).done(function(response) {
+        addHTML(response);
+      });
+    });
+  };
+  
+  $(document).ready(function(){
+    bindCreateButton();
+  });
+```
+Here we are sending markdown to the GitHub API to render into HTML. Once we get a response it adds the HTML to `<div id="search_results">`.
```

Comment:
awesome - makes total sense. maybe add in some expo about the API endpoint then, that you can't visit the route in the browser because it only accepts POSTs just so students don't have the same experience i did